.github/workflows/bigdl-llm-winlog.yml

```diff
@@ -0,0 +1,22 @@
+name: bigdl-llm-win
+
+on:
+  # push:
+  #   paths:
+  #     - 'python/llm'
+  #     - '.github/workflows/bigdl-llm-winlog.yml'
+  # pull_request:
+  #   branchs: [ main ]
+  schedule:
+    - cron: '0 3 * * *' # GMT time, 3:00 GMT == 3:00 China Everyday
+  workflow_dispatch:
+
+jobs:
+  bigdl-llm-win:
+    runs-on: [self-hosted, Gondolin, ubuntu-20.04-lts]
+    steps:
+      - name: Download log
+        run: |
+          wget ftp://10.112.231.51:8821/llm/win_llm_test.log --ftp-user=${{ secrets.FTP_USERNAME }} --ftp-password=${{ secrets.FTP_PASSWORD }}
+          cat win_llm_test.log
```

Comment:
so basically we need to manual check if there is any error or not?

Reply:
Yes.